plone/app/discussion/profiles/default/catalog.xml

```diff
@@ -14,5 +14,6 @@
     <column value="commentators" />
     <column value="in_response_to" />
     <column value="author_name" />
+    <column value="author_email" />
```

Comment:
It's  bit unfortunate that a new metadata column is needed, as this will be added for all and every single object, not just for comments.

This also means that as soon as Plone is updated to use this p.a.discussion version, all content (at least the comments) need to be reindex, if not the moderation view will fail, right?

Now I see the ``tal:condition`` is that enough to not make the view fail if a comment does not have a have an email?